inst/rmd_web/index.Rmd

```diff
@@ -11,7 +11,7 @@ output:
     
 ---
 
-Last updated `r Sys.time()`
+Last updated `r Sys.time()` CDT
```

Comment:
Did you check that the servers are set to CDT? You can use `Sys.timezone()` to check the timezone R and the computer are using.